site/app/templates/grading/electronic/GradingPanelHeader.twig

```diff
@@ -56,6 +57,15 @@
             {{ self.renderPanelPositionSelector('student_info') }}
             </span>
         {% endif %}
+        <span class="grade-panel" id="solution_ta_notes_btn">
```

Comment:
HTML CSS Id's should use dashes not underscores

https://submitty.org/developer/coding_style_guide/css